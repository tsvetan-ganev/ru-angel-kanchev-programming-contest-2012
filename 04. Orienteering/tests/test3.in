20 2
1802Zhelyazkov m 09:24:48 10:13:32
0402Atanasov m 10:09:09 10:51:23
2002Noev m 10:50:32 11:39:16
1301Simeonov m 08:18:30 09:36:48
1701Iliev m 10:24:03 11:24:34
1601Vasilev m 10:43:46 11:48:16
1901Vasilev m 10:05:29 11:07:14
1502Damyanov m 10:34:03 11:30:41
0502Kostadinov m 09:03:37 10:20:15
1201Karavasilev m 08:39:47 09:35:44
0601Mihaylov m 10:24:31 11:19:22
0202Georgiev m 10:06:41 11:17:20
0302Antonov m 10:46:28 11:49:13
0702Nankov m 09:48:38 10:44:16
0101Milev m 10:25:11 11:32:38
1002Venev m 09:11:14 10:10:48
1402Ponev m 08:02:33 08:47:15
0901Marinov m 08:32:34 09:35:55
0802Sofin m 08:25:43 09:27:04
1102Stefanov m 09:01:01 10:10:31