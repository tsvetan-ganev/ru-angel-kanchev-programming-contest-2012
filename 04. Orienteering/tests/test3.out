04 14 18 20 

02 